addons/core/stringtable.xml

```diff
@@ -7,22 +7,26 @@
                 <Original>Task Force Arrowhead Radio Team</Original>
                 <English>Task Force Arrowhead Radio Team</English>
                 <Japanese>タスクフォース アローヘッド ラジオ チーム</Japanese>
+                <Korean>Task Force Arrowhead Radio Team</Korean>
             </Key>
             <Key ID="STR_TFAR_CORE_CATEGORY">
                 <Original>TFAR</Original>
                 <English>TFAR</English>
                 <Japanese>TFAR</Japanese>
+                <Korean>TFAR</Korean>
```

Comment:
Don't add Translations if they equal the `Original` entry.
Japanese here should be removed too.